src/status_im/constants.cljs

```diff
@@ -21,7 +21,7 @@
 (def max-chat-name-length 20)
 (def max-group-chat-participants 10)
 (def response-suggesstion-resize-duration 100)
-(def default-number-of-messages 20)
+(def default-number-of-messages 10)
```

Comment:
that might have had an impact on perfs, though that might not be enough for some big displays

Reply:
oh I actually forgot to change it back, I was testing something

Reply:
tested on One Plus Pro 7 there is a clear jump when you open a chat with small messages and more than 10 fit in one screen

Reply:
sorry our messages crossed